pkgs/os-specific/linux/systemd/default.nix

```diff
@@ -281,9 +281,6 @@ stdenv.mkDerivation {
     "-Dmount-path=${utillinux}/bin/mount"
     "-Dumount-path=${utillinux}/bin/umount"
     "-Dcreate-log-dirs=false"
-    # Upstream uses cgroupsv2 by default. To support docker and other
-    # container managers we still need v1.
-    "-Ddefault-hierarchy=hybrid"
```

Comment:
Ugh, why not be explicit? Both here and via the module system. If we have this in the module system we can throw proper warnings/errors when docker is enabled.

Reply:
Makes sense, done.